src/Ben.Demystifier/EnhancedStackTrace.Frames.cs

```diff
@@ -69,6 +69,11 @@ private static List<EnhancedStackFrame> GetFrames(StackTrace stackTrace)
                         portablePdbReader.PopulateStackFrame(frame, method, frame.GetILOffset(), out fileName, out row, out column);
                     }
 
+                    if (method is null)
```

Comment:
Without this I would need to mark downstream args as nullable and lots of usages of them would light up as potential null reference exception. This seems the way to go